source/MaterialXRuntime/RtValue.cpp

```diff
@@ -59,34 +59,44 @@ RtValue RtValue::createNew(const RtToken& type, RtObject owner)
     return typeDef->createValue(owner);
 }
 
-void RtValue::toString(const RtToken& type, const RtValue& src, string& dest)
+void RtValue::copy(const RtToken& type, const RtValue& src, RtValue& dest)
```

Comment:
The diff of this file came out very strange. The only change done is the addition of RtValue::compare()